plugins/header_rewrite/operators.cc

```diff
@@ -279,10 +280,9 @@ OperatorSetRedirect::initialize(Parser &p)
 
   _status.set_value(p.get_arg());
   _location.set_value(p.get_value());
-
-  if ((_status.get_int_value() != (int)TS_HTTP_STATUS_MOVED_PERMANENTLY) &&
-      (_status.get_int_value() != (int)TS_HTTP_STATUS_MOVED_TEMPORARILY)) {
-    TSError("[%s] unsupported redirect status %d", PLUGIN_NAME, _status.get_int_value());
+  auto status = _status.get_int_value();
```

Comment:
It would be better to specify the response codes.  304 Not Modified is not a redirect and most values between 300 and 399 are not valid codes.

Reply:
I talked to @zwoop about this specific point and he felt it better to not embed that in the plugin. I can fix it if you think it's actually important.